python/CMakeLists.txt

```diff
@@ -1,43 +1,26 @@
 cmake_minimum_required(VERSION 3.18)
 
-project(fenics-dolfinx)
+project(dolfinx_nanobind)
 
-# Set C++ standard before finding pybind11
+# Set C++ standard
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
-find_package(
-  Python3
-  COMPONENTS Interpreter Development.Module
-  REQUIRED
-)
+# set(CMAKE_BUILD_TYPE Debug)
```

Comment:
I guess this commented line can be removed, since the default `CMAKE_BUILD_TYPE` is set anyways two lines afterwards.